models/User.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+
+const UserSchema = new mongoose.Schema({
+    name: {
+        type: String,
+        required: [true, 'Please provide the name'],
+        minLength: 3,
+        maxLength: 30,
+    },
+    email: {
+        type: String,
+        required: [true, 'Please provide the name'],
```

Comment:
I think you want this message to be `'Please provide the email'`

Reply:
Thanks. Fixed!